sling/nlp/wiki/wiki.h

```diff
@@ -24,6 +24,13 @@
 namespace sling {
 namespace nlp {
 
+// Wikidata object types.
+enum WikidataType {
+  WIKIDATA_ITEM = 0,
+  WIKIDATA_PROPERTY = 1,
+  WIKIDATA_LEXEME = 1,
```

Comment:
Suppose WIKIDATA_LEXEME should be equal to 2.

Reply:
Oops, that's a bug. Fixed. I did not notice this during my test because my Wikidata dump does not have any lexemes.